db/re/item_db.txt

```diff
@@ -1308,6 +1309,7 @@
 2030,Unity_Two-Handed_Staff,Unity Two-Handed Staff,5,20,,500,95:135,,1,1,0x800200,63,2,34,3,1,1,23,{ bonus bMatk,pow(getrefine(),2)*125/100; },{},{}
 2032,Requiem_Wizardry_Staff,Requiem Wizardry Staff,5,20,,500,100:230,,1,,0xFFFFFFFF,63,2,2,4,100,1,10,{ .@r = getrefine(); .@b = 40; .@c = 5; if(.@r>=5){ .@b += (.@r-5)*15+20; .@c += .@r-5; } bonus2 bMagicAddRace2,RC2_BioLab,.@b; bonus2 bSubRace2,RC2_BioLab,.@c; bonus2 bIgnoreMdefRace2Rate,RC2_BioLab,(.@r-5)*10; },{},{}
 2039,Illusion_Wizardry_Staff,Illusion Wizardry Staff,5,0,,2400,150:250,,1,2,0x00810204,58,2,34,4,100,1,23,{ .@val = min(getrefine(),10)/2; bonus bInt,(6+.@val); bonus bDex,(2+.@val); },{},{}
+2048,Aeon_Staff,Aeon Staff,5,,,800,100:200,,1,2,0x00000200,56,2,34,4,100,1,23,{ bonus bMatk,200; bonus bUnbreakableWeapon; .@r = getrefine(); if (.@r >= 2) bonus bMatk,.@r/2*10; if (.@r >= 3) bonus2 bSkillAtk,"WL_EARTHSTRAIN",.@r/3*12; if (.@r >= 2) { bonus bVariableCastrate,-10; } if (.@r >= 2) { bonus2 bSkillCooldown,"WL_EARTHSTRAIN",-1000; } },{},{}
```

Comment:
This script should be 

.@r = getrefine; bonus bMatk,10*.@r/2; bonus2 bSkillAtk,"WL_EARTHSTRAIN",12*.@r/3; if (.@r>=9) bonus bVariableCastrate,-10; if (.@r>=11) bonus2 bSkillCooldown,"WL_EARTHSTRAIN",-1000; bonus bUnbreakableWeapon;